pyccel/parser/semantic.py

```diff
@@ -902,7 +902,20 @@ def _handle_function_args(self, arguments, **settings):
                 args.append(a)
         return args
 
-    def _handle_function(self, func, args, **settings):
+    def _handle_function(self, expr, func, args, **settings):
+        """
+        Create a FunctionCall or an instance of a PyccelInternalFunction
+        from the function informations and arguments
+
+        Parameters
+        ==========
+        expr : PyccelAstNode
+               The expression where this call is found (used for error output)
+        func : FunctionDef, Interface or PyccelInternalFunction type
+               The function being called
+        args : tuple
+               The arguments passed to the function
+        """
```

Comment:
Thanks! But I think the "Returns" section is missing...

Reply:
Fixed in 47de259